source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.h

```diff
@@ -63,7 +63,7 @@ class SwiftASTManipulatorBase {
 
     VariableInfo(CompilerType &type, swift::Identifier name,
                  VariableMetadataSP metadata, 
-                 swift::VarDecl::Specifier specifier = swift::VarDecl::Specifier::Let,
+                 swift::VarDecl::Specifier specifier = swift::VarDecl::Specifier::Var,
```

Comment:
The code that used to be here was

```
bool is_let = false
```

I just translated it to the new representation.